RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernels.cu

```diff
@@ -270,11 +270,13 @@ void CAHitNtupletGeneratorKernelsGPU::classifyTuples(HitsOnCPU const &hh, TkSoA
     kernel_fillHitInTracks<<<numberOfBlocks, blockSize, 0, cudaStream>>>(tuples_d, quality_d, device_hitToTuple_.get());
     cudaCheck(cudaGetLastError());
   }
-  if (m_params.minHitsPerNtuplet_ < 4) {
+
+  if (m_params.doSharedHitCut_)
```

Comment:
line 262 need to be fixed as well

Reply:
please check all "m_params.minHitsPerNtuplet_ < 4" or "m_params.minHitsPerNtuplet_ >=3" and similar